src/libsystemd-network/dhcp-lease-internal.h

```diff
@@ -75,6 +75,8 @@ struct sd_dhcp_lease {
         uint16_t mtu; /* 0 if unset */
 
         char *domainname;
+        char **search_domains;
+        size_t search_domains_count;
```

Comment:
`search_domains_count` doesn't seem to be used for anything. Just drop it. You can check if `search_domains` is empty with `strv_empty()`.